.github/workflows/prod_deploy.yaml

```diff
@@ -1,18 +1,19 @@
-name: Workflow 2 - SAM Validate, Build, Test, Deploy
+name: Workflow 2 - SAM Validate, Build, Deploy
 on:
-  repository_dispatch:
-    types: [trigger-workflow-2]
+  workflow_run:
+    workflows: ["Workflow 1 - Test"]
+    branches: [master]
+    types:
+      - completed
```

Comment:
Maybe this could be called "Workflow 1 - Run Test Suite"?  When I saw "Workflow 1 - Test" it just felt like it was temporary.

Reply:
fixed!